realm/src/main/java/io/realm/Realm.java

```diff
@@ -114,12 +114,15 @@
     private static final String APT_NOT_EXECUTED_MESSAGE = "Annotation processor may not have been executed.";
     private static final String INCORRECT_THREAD_MESSAGE = "Realm access from incorrect thread. Realm objects can only be accessed on the thread they where created.";
     private static final String CLOSED_REALM = "This Realm instance has already been closed, making it unusable.";
+    private static final String INVALID_KEY_MESSAGE = "The provided key is illegal. It should either be null or be 64 bytes long";
```

Comment:
This sentence could be clearer i think.  Perhaps rephrase to "Illegal key length. It should either be null or 64 bytes long"